user/tests/wiring/ble_central_peripheral/ble_central/central.cpp

```diff
@@ -438,7 +442,7 @@ static void pairingTestRoutine(bool request, BlePairingAlgorithm algorithm,
             assertTrue(waitFor([&]{ return pairingRequested; }, 5000));
         }
         assertTrue(BLE.isPairing(peer));
-        assertTrue(waitFor([&]{ return !BLE.isPairing(peer); }, 60000));
+        assertTrue(waitFor([&]{ return !BLE.isPairing(peer); }, 20000));
```

Comment:
@XuGuohui Can we leave this high just in case?

Reply:
It is probably changed to 6000 since last commit for debugging purpose and is reverted in next commit. It had passed the tests anyway after the final change.